Description: Make build reproducible
Author: Reiner Herrmann <reiner@reiner-h.de>
Forwarded: https://bugs.launchpad.net/onboard/+bug/1530519

Index: onboard-1.1.2/setup.py
===================================================================
--- onboard-1.1.2.orig/setup.py
+++ onboard-1.1.2/setup.py
@@ -72,7 +72,7 @@ def pkgconfig(*packages, **kw):
         else:
             kw.setdefault('extra_link_args', []).append(token)
     for k, v in kw.items():
-        kw[k] = list(set(v))
+        kw[k] = sorted(list(set(v)))
 
     return kw
 
