Description: Fix build with GCC 6
Forwarded: https://bazaar.launchpad.net/~onboard/onboard/trunk/revision/2155
Origin: https://git.archlinux.org/svntogit/community.git/tree/trunk/gcc6.diff?h=packages/onboard
Bug-Debian: https://bugs.debian.org/812033

--- a/Onboard/pypredict/lm/lm_dynamic_cached.h
+++ b/Onboard/pypredict/lm/lm_dynamic_cached.h
@@ -224,10 +224,11 @@
     public:
         typedef _DynamicModelKN<TNGRAMS> Base;
         static const Smoothing DEFAULT_SMOOTHING = ABS_DISC_I;
-        static const double DEFAULT_LAMBDA = 0.3;  // Jelinek-Mercer weights
+        const double DEFAULT_LAMBDA;  // default for Jelinek-Mercer weights
 
     public:
-        _CachedDynamicModel()
+        _CachedDynamicModel() :
+            DEFAULT_LAMBDA(0.3)
         {
             this->smoothing = DEFAULT_SMOOTHING;
             recency_smoothing = JELINEK_MERCER_I;
