Origin: https://bazaar.launchpad.net/~onboard/onboard/trunk/revision/2129
Bug: https://launchpad.net/bugs/1567301
Index: onboard-1.2.0/settings.ui
===================================================================
--- onboard-1.2.0.orig/settings.ui	2016-02-18 14:31:51.000000000 +0100
+++ onboard-1.2.0/settings.ui	2016-04-15 21:27:35.187583393 +0200
@@ -273,7 +273,7 @@
             <property name="layout_style">end</property>
             <child>
               <object class="GtkButton" id="close_button">
-                <property name="label">_Close</property>
+                <property name="label" translatable="yes">_Close</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="can_default">True</property>
@@ -1458,7 +1458,7 @@
                         <property name="layout_style">start</property>
                         <child>
                           <object class="GtkButton" id="layout_new_button">
-                            <property name="label">_New</property>
+                            <property name="label" translatable="yes">_New</property>
                             <property name="visible">True</property>
                             <property name="can_focus">True</property>
                             <property name="receives_default">True</property>
@@ -1473,7 +1473,7 @@
                         </child>
                         <child>
                           <object class="GtkButton" id="layout_remove_button">
-                            <property name="label">_Delete</property>
+                            <property name="label" translatable="yes">_Delete</property>
                             <property name="visible">True</property>
                             <property name="can_focus">True</property>
                             <property name="receives_default">True</property>
@@ -1503,7 +1503,7 @@
                         </child>
                         <child>
                           <object class="GtkButton" id="layout_about_button">
-                            <property name="label">_About</property>
+                            <property name="label" translatable="yes">_About</property>
                             <property name="visible">True</property>
                             <property name="can_focus">True</property>
                             <property name="receives_default">True</property>
@@ -1613,7 +1613,7 @@
                             <property name="layout_style">start</property>
                             <child>
                               <object class="GtkButton" id="new_theme_button">
-                                <property name="label">_New</property>
+                                <property name="label" translatable="yes">_New</property>
                                 <property name="visible">True</property>
                                 <property name="can_focus">True</property>
                                 <property name="receives_default">True</property>
@@ -1628,7 +1628,7 @@
                             </child>
                             <child>
                               <object class="GtkButton" id="delete_theme_button">
-                                <property name="label">_Delete</property>
+                                <property name="label" translatable="yes">_Delete</property>
                                 <property name="visible">True</property>
                                 <property name="can_focus">True</property>
                                 <property name="receives_default">True</property>
@@ -1777,7 +1777,7 @@
                         <property name="layout_style">start</property>
                         <child>
                           <object class="GtkButton" id="snippet_add_button">
-                            <property name="label">_Add</property>
+                            <property name="label" translatable="yes">_Add</property>
                             <property name="visible">True</property>
                             <property name="can_focus">True</property>
                             <property name="receives_default">True</property>
@@ -1792,7 +1792,7 @@
                         </child>
                         <child>
                           <object class="GtkButton" id="snippet_remove_button">
-                            <property name="label">_Remove</property>
+                            <property name="label" translatable="yes">_Remove</property>
                             <property name="visible">True</property>
                             <property name="can_focus">True</property>
                             <property name="receives_default">True</property>
Index: onboard-1.2.0/settings_auto_show_dialog.ui
===================================================================
--- onboard-1.2.0.orig/settings_auto_show_dialog.ui	2016-02-18 14:31:51.000000000 +0100
+++ onboard-1.2.0/settings_auto_show_dialog.ui	2016-04-15 21:32:07.251576063 +0200
@@ -81,7 +81,7 @@
             <property name="layout_style">end</property>
             <child>
               <object class="GtkButton" id="dialog_close_button">
-                <property name="label">_Close</property>
+                <property name="label" translatable="yes">_Close</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="receives_default">True</property>
Index: onboard-1.2.0/settings_docking_dialog.ui
===================================================================
--- onboard-1.2.0.orig/settings_docking_dialog.ui	2016-02-18 14:31:51.000000000 +0100
+++ onboard-1.2.0/settings_docking_dialog.ui	2016-04-15 21:35:13.779571038 +0200
@@ -89,7 +89,7 @@
             <property name="layout_style">end</property>
             <child>
               <object class="GtkButton" id="dialog_close_button">
-                <property name="label">_Close</property>
+                <property name="label" translatable="yes">_Close</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="receives_default">True</property>
Index: onboard-1.2.0/settings_scanner_dialog.ui
===================================================================
--- onboard-1.2.0.orig/settings_scanner_dialog.ui	2016-02-18 14:31:51.000000000 +0100
+++ onboard-1.2.0/settings_scanner_dialog.ui	2016-04-15 21:37:55.915566670 +0200
@@ -92,7 +92,7 @@
             <property name="layout_style">end</property>
             <child>
               <object class="GtkButton" id="dialog_close_button">
-                <property name="label">_Close</property>
+                <property name="label" translatable="yes">_Close</property>
                 <property name="use_action_appearance">False</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
Index: onboard-1.2.0/settings_theme_dialog.ui
===================================================================
--- onboard-1.2.0.orig/settings_theme_dialog.ui	2016-02-18 14:31:51.000000000 +0100
+++ onboard-1.2.0/settings_theme_dialog.ui	2016-04-15 21:51:52.055544142 +0200
@@ -989,7 +989,7 @@
             <property name="layout_style">end</property>
             <child>
               <object class="GtkButton" id="revert_button">
-                <property name="label">_Revert</property>
+                <property name="label" translatable="yes">_Revert</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="receives_default">False</property>
@@ -1004,7 +1004,7 @@
             </child>
             <child>
               <object class="GtkButton" id="close_button">
-                <property name="label">_Close</property>
+                <property name="label" translatable="yes">_Close</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="can_default">True</property>
