Description: Add Keywords= field, drop MimeType= fields.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/data/onboard-settings.desktop.in
+++ b/data/onboard-settings.desktop.in
@@ -6,6 +6,6 @@
 Terminal=false
 Type=Application
 Categories=Settings;Accessibility;
-MimeType=application/x-onboardsettings;
+_Keywords=onscreen;keyboard;accessibility;utility;settings;configuration;
 Icon=onboard
 X-Ubuntu-Gettext-Domain=onboard
--- a/data/onboard.desktop.in
+++ b/data/onboard.desktop.in
@@ -6,6 +6,6 @@
 Terminal=false
 Type=Application
 Categories=Utility;Accessibility;
-MimeType=application/x-onboard;
+_Keywords=onscreen;keyboard;accessibility;utility;
 Icon=onboard
 X-Ubuntu-Gettext-Domain=onboard
